includes/apple-exporter/class-theme.php

```diff
@@ -697,55 +750,39 @@ private static function initialize_options() {
 			),
 			'byline_color'                       => array(
 				'default' => '#7c7c7c',
-				'label'   => __( 'Byline font color', 'apple-news' ),
+				'label'   => __( 'Unified Byline font color', 'apple-news' ),
 				'type'    => 'color',
 			),
 			'byline_color_dark'                  => array(
 				'default' => '',
-				'label'   => __( 'Byline font color', 'apple-news' ),
+				'label'   => __( 'Unified Byline font color', 'apple-news' ),
 				'type'    => 'color',
 			),
 			'byline_font'                        => array(
 				'default' => 'AvenirNext-Medium',
-				'label'   => __( 'Byline font face', 'apple-news' ),
+				'label'   => __( 'Unified Byline font face', 'apple-news' ),
 				'type'    => 'font',
 			),
 			'byline_format'                      => array(
 				'default'     => 'by #author# | #M j, Y | g:i A#',
 				'description' => __( 'Set the byline format. Two tokens can be present, #author# to denote the location of the author name and a <a href="http://php.net/manual/en/function.date.php" target="blank">PHP date format</a> string also encapsulated by #. The default format is "by #author# | #M j, Y | g:i A#". Note that byline format updates only preview on save.', 'apple-news' ),
-				'label'       => __( 'Byline format', 'apple-news' ),
+				'label'       => __( 'Unified Byline format', 'apple-news' ),
 				'type'        => 'text',
 			),
-			'byline_link_color'                  => array(
-				'default' => '#7c7c7c',
-				'label'   => __( 'Byline link font color', 'apple-news' ),
-				'type'    => 'color',
-			),
-			'byline_link_color_dark'             => array(
-				'default' => '',
-				'label'   => __( 'Byline link font color ', 'apple-news' ),
-				'type'    => 'color',
-			),
-			'byline_links'                       => array(
-				'default' => 'no',
-				'label'   => __( 'Byline author links', 'apple-news' ),
-				'options' => array( 'yes', 'no' ),
-				'type'    => 'select',
-			),
```

Comment:
Normally, I would say that these shouldn't be deleted, but the byline link feature hasn't been released yet, so we can refactor these into generic settings that apply to both the byline and the author component.

Reply:
These were deleted here in favor of the `author_link...` settings for that exact reason. 👍  APPLE-71 was the above, and this ticket refactored that to be used in the Author and Byline components.